### PR TITLE
add pauseTransfer flag option to rules

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Closes https://github.com/jbx-protocol/juice-interface/issues/2842

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2876** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2876" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2876?utm_source=stack-comment).

File: src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/FundingAttributes.tsx [hunk 1]
```diff
@@ -308,6 +308,28 @@ export function AllowSetTerminalsStatistic({
...
+  pauseTransfers: boolean
+}) {
+  return (
+    <Statistic
+      title={
+        <TooltipLabel
+          label={t`Pause transfers`}
+          tip={
+            pauseTransfers
+              ? t`Transfers are paused`
```
Comment (mejango) [inline src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/FundingAttributes.tsx:323]: "Project token transfers are paused/unpaused. This does not apply to ERC-20 tokens if issued."

File: src/components/v2v3/shared/FundingCycleConfigurationDrawers/RulesDrawer/RulesForm/RulesForm.tsx [hunk 2]
```diff
@@ -161,6 +169,20 @@ export default function RulesForm({
...
+          >
+            <div className="flex font-medium text-black dark:text-slate-100">
+              <Switch
+                className="mr-2"
+                onChange={checked => {
+                  setPauseTransfers(checked)
+                }}
+                checked={pauseTransfers}
+              />
+              <Trans>Pause transfers</Trans>
```
Comment (mejango) [inline src/components/v2v3/shared/FundingCycleConfigurationDrawers/RulesDrawer/RulesForm/RulesForm.tsx:183]: Pause project token transfers. Does not apply to ERC-20 tokens if issued.

File: src/locales/messages.pot [hunk 3]
```diff
@@ -3200,6 +3203,9 @@ msgstr ""
 msgid "Transfer {tokenTextShort}"
 msgstr ""
 
+msgid "Transfers are paused"
```
Comment (mejango) [inline src/locales/messages.pot:3206]: Transfers of project tokens are paused. This does not apply to ERC-20 tokens if issued.

File: src/locales/messages.pot [hunk 4]
```diff
@@ -3422,6 +3428,9 @@ msgstr ""
 msgid "When distributing, payouts to Ethereum addresses incur a 2.5% JBX membership fee. Payouts to other Juicebox projects don't incur fees. Your project will receive (the <0>JuiceboxDAO</0> token) in return at the current issuance rate."
 msgstr ""
 
+msgid "When enabled, all transfers will be paused."
```
Comment (mejango) [inline src/locales/messages.pot:3431]: transfers to project tokens will be paused. This does not apply to ERC-20 tokens if issued.

File: src/locales/messages.pot [hunk 5]
```diff
@@ -3437,6 +3446,9 @@ msgstr ""
 msgid "When enabled, the project will hold the fee amount in ETH instead of the fees being processed automatically. <0>Learn more.</0>"
 msgstr ""
 
+msgid "When enabled, transfers will be paused."
```
Comment (mejango) [inline src/locales/messages.pot:3449]: ...project token transfers will be paused. This does not apply to ERC-20 tokens if issued.

File: src/components/Create/components/pages/ReconfigurationRules/ReconfigurationRulesPage.tsx [hunk 6]
```diff
@@ -76,6 +76,12 @@ export const ReconfigurationRulesPage = () => {
             >
               <JuiceSwitch label={t`Allow terminal configuration`} />
             </Form.Item>
+            <Form.Item
+              name="pauseTransfers"
+              extra={t`When enabled, transfers will be paused.`}
```
Comment (mejango) [inline src/components/Create/components/pages/ReconfigurationRules/ReconfigurationRulesPage.tsx:81]: ... project token transfers will be paused. This does not apply to ERC-20 tokens if issued.

**Review by mejango — CHANGES_REQUESTED** [review]

**Review by johnnyd-eth — APPROVED** [review]
Tested this and it's all working well, and code looks great. Will merge!

**Pull request merged**